charts/node-server/templates/mongodb-deploy.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb
+  namespace: "tensorleap"
```

Comment:
```suggestion
```

Reply:
Fixed
What does this "namespace" prop do?